src/templates/fr/2020/index.html

```diff
@@ -1,24 +1,24 @@
 {% extends "base/2020/index.html" %}
 
-{% block title %}Web Almanac 2019{% endblock %}
+{% block title %}Web Almanac {{ year }}{% endblock %}
 {% block description %}Le Web Almanac est un rapport annuel sur l’état du Web qui combine l’expertise de la communauté Web avec les données et tendances de HTTP Archive.{% endblock %}
 
 {% block twitter_image_alt %}Web Almanac {{ year }}{% endblock %}
 
 {% block date_published %}2019-11-04T12:00:00.000Z{% endblock %}
 {% block date_modified %}2020-07-06T00:00:00.000Z{% endblock %}
 
-{% block intro_title %}Coming soon!{% endblock %}
-{% block intro_sub_title %}The 2020 <b>state of the web</b> report{% endblock %}
+{% block intro_title %}Bientôt disponible&nbsp;!{% endblock %}
+{% block intro_sub_title %}Le rapport 2020 sur <b>l’état du Web</b>{% endblock %}
```

Comment:
If you're gonna use the `{{ year }}` variable above (which I did suggest to Rick btw!) then might as well use it here too:

```suggestion
{% block intro_sub_title %}Le rapport {{ year }} sur <b>l’état du Web</b>{% endblock %}
```